pages/common/jdeps.md

```diff
@@ -0,0 +1,24 @@
+# jdeps
+
+> Java Class Dependency Analyzer.
+> More information: <https://docs.oracle.com/en/java/javase/17/docs/specs/man/jdeps.html>.
+
+- Analyze the dependencies of a `.jar` or `.class` file:
+
+`jdeps {{path/to/filename.class}}`
+
+- Print a summary of all dependencies:
+
+`jdeps {{path/to/filename.jar}} -summary`
+
+- Print all class-level dependencies:
+
+`jdeps {{filename.jar}} -verbose`
```

Comment:
```suggestion
`jdeps {{path/to/filename.jar}} -verbose`
```

Reply:
Ah oops, I'll double check the entire page again quickly then add that suggestion too lol